hickle/__init__.py

```diff
@@ -2,10 +2,10 @@
 from .__version__ import __version__
 from . import hickle
 from .hickle import *
+from .fileio import ClosedFileError, FileError
 
 # All declaration
-__all__ = ['hickle']
-__all__.extend(hickle.__all__)
+__all__ = ['hickle', *hickle.__all__]
```

Comment:
Revert this line back to what it used to be.
It makes it less clear that this list is made up of basic items and items of other lists.